src/validators/block_id/mod.rs

```diff
@@ -103,26 +103,40 @@ pub fn display_current_validators_info(
     );
 
     let mut table = Table::new();
-    table.set_titles(prettytable::row![Fg=>"Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected"]);
+    table.set_titles(prettytable::row![Fg=>"#", "Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected", "Endorsements produced", "Endorsements expected"]);
 
-    for validator in &current_validators {
+    for (index, validator) in current_validators.into_iter().enumerate() {
         let online = if validator.num_expected_blocks + validator.num_expected_chunks == 0 {
+            if validator.num_expected_endorsements == 0 {
+                "NaN".to_string()
+            } else {
+                format!(
+                    "{:>6.2} %",
+                    (validator.num_produced_endorsements * 100) as f64
+                        / (validator.num_expected_endorsements) as f64
+                )
+            }
+        } else if validator.num_expected_blocks == 0 || validator.num_expected_chunks == 0 {
```

Comment:
If any of these is 0, it should be just ignored (if num_blocks_expected => return num_chunks / num_chunks_expected, and similarly for num_chunks_expected == 0)

Reply:
I use this formula but want to avoid dividing by 0
<img width="773" alt="Screenshot 2024-08-16 at 21 49 10" src="https://github.com/user-attachments/assets/0f88483d-e7c3-486a-ae05-1ccad6376e0f">

Reply:
Sure, but you should avoid division by 0 in a reasonable way. Once a new epoch comes, the expected blocks/chunks produced will be 0 (and I'd assume 100% online until it is clearly not), and soon it will get its first chunk OR block expected and if it was produced (1 block produced and expected while 0 chunks expected), it should be still reported as 100% online, not NaN.

Reply:
Done